packages/explorer/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["../../packages/common/tsconfig.base.json"],
   "compilerOptions": {
+    "lib": ["DOM", "ESNext"],
```

Comment:
this is already in the base config afaik

Reply:
nice catch! removed, couldn't spot any more redundant ones